### DISCUSSION TITLE
frame using buttom transaction

### DISCUSSION THREAD

**Discussion by Chinaskidev** [wevm/frog-archived#475] [General]
hi, everyone, im trying to do a frame using buttom. transaction, im new in this world of dev, and.. i did a  smart contract in thirdweb, and im trying to use test my frame with my wallet but i have a server error 500 i dont know why? in my terminal i can see the address of my wallet. S
![Captura de pantalla de 2024-08-22 15-46-57](https://github.com/user-attachments/assets/cc4a5801-cb79-4aef-a803-affea871e0e7)
omebody wants to show the way? jeje i wil put a image maybe you can see my errors. thanks a cheers!!!
![Captura de pantalla de 2024-08-22 15-44-53](https://github.com/user-attachments/assets/9d46c04a-90e5-4c46-9a58-df4898a4f1da)
![Captura de pantalla de 2024-08-22 15-45-37](https://github.com/user-attachments/assets/2d872d95-a9b2-4812-bd85-85c79f2be837)

**Comment by dalechyn** [comment]
Hey, sorry for a late response. `500` status code is a server error, so you should normally see logs in your console.

Otherwise please create an issue with minimal reproducible example, it's hard to understand cause of the issue here.
